src/todo/todo.ts

```diff
@@ -0,0 +1,32 @@
+import { FilterBy, Todo } from '../types/Todo';
+
+export function filterGoods(
+  todos: Todo[],
+  filterBy: FilterBy,
+  query: string,
+) {
+  let copyTodos = [...todos];
```

Comment:
The filter method returns a copy of the array, so you don't need to do it manually.